front/lib/api/assistant/configuration.ts

```diff
@@ -814,23 +723,11 @@ export async function createAgentConfiguration(
               transaction: t,
             }
           );
+          userFavorite = userRelation?.favorite ?? false;
         }
+
         const sId = agentConfigurationId || generateLegacyModelSId();
 
-        // If creating a new agent, we include it in the user's list by default.
-        // This is so it doesn't disappear from their list on scope change
-        if (!agentConfigurationId) {
-          listStatusOverride = "in-list";
-          await AgentUserRelation.create(
```

Comment:
Without this will I still be able to create an assistant and then start using it right away?